.github/workflows/cla.yml

```diff
@@ -1,4 +1,4 @@
-name: Verify Contributor License Agreement
+name: CLA Check
```

Comment:
Just curious, between this is and the labeler name, is there a reason you're preferring shorter names?

Reply:
> Just curious, between this is and the labeler name, is there a reason you're preferring shorter names?

It's just a minor UI quality of life detail. In Github UI, these are truncated

![image](https://github.com/user-attachments/assets/1779bd7e-abcf-41f9-b3cc-ec75a44d8eaa)

![image](https://github.com/user-attachments/assets/09b89d02-1573-4dc5-bd89-8c4bef5960ba)

Reply:
> It's just a minor UI quality of life detail. In Github UI, these are truncated
> ...

I think that the following is slightly better as a result. With this change, it reduces truncation and increases signal to noise ratio by eliminating redundant data.

![image](https://github.com/user-attachments/assets/c256526c-5468-4d5f-b4a9-04f3728d5b6c)

Reply:
Makes sense, thanks!